data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -196,3 +197,47 @@
 HOLC_GRADE_D_TRACT_20_PERCENT_FIELD: str = "Tract is >20% HOLC Grade D"
 HOLC_GRADE_D_TRACT_50_PERCENT_FIELD: str = "Tract is >50% HOLC Grade D"
 HOLC_GRADE_D_TRACT_75_PERCENT_FIELD: str = "Tract is >75% HOLC Grade D"
+
+
+# Climate Change
+EXPECTED_POPULATION_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected population loss rate and is low income"
+EXPECTED_AGRICULTURE_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected agriculture loss rate and is low income"
+EXPECTED_BUILDING_LOSS_RATE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for expected building loss rate and is low income"
+
+# Clean energy and efficiency
+PM25_EXPOSURE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for PM2.5 exposure and is low income"
+ENERGY_BURDEN_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for energy burden and is low income"
+# Clean transportation
+DIESEL_PARTICULATE_MATTER_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for diesel particulate matter and is low income"
+TRAFFIC_PROXIMITY_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for traffic proximity and is low income"
+# Affordable and Sustainable Housing
+LEAD_PAINT_MEDIAN_HOME_VALUE_LOW_INCOME_FIELD = f"At or above the {PERCENTILE}th percentile for lead paint and the median house value is less than {PERCENTILE}th percentile and is low income"
```

Comment:
Sorry for not catching this sooner -- please create a new MEDIAN_HOUSE_VALUE_PERCENTILE = 90. These happen to be the same thing but they may drift to different values. 

So the whole value would be: 

```
MEDIAN_HOUSE_VALUE_PERCENTILE = 90

...

LEAD_PAINT_MEDIAN_HOME_VALUE_LOW_INCOME_FIELD =  f"At or above the {PERCENTILE}th percentile for lead paint and the median house value is less than {MEDIAN_HOUSE_VALUE_PERCENTILE}th percentile and is low income"

Reply:
Gotcha